README.md

```diff
@@ -17,61 +17,64 @@ This [Snap](https://snapcraft.io/) is part the
+  - AMD64 dual-core processor around 2GHz clock speed or faster
+  - 4GB RAM
+  - 32GB or greater SSD storage
+- At least two ethernet interfaces (SGi and S1)
 - Ubuntu 20.04 LTS installed
   ([Ubuntu installation guide](https://help.ubuntu.com/lts/installation-guide/amd64/index.html))
 
 ## Usage
 
-This Snap is not yet available in the official [Snap store](https://snapcraft.io/store), which
-means it needs to be built locally on user's hardware.<br>
+This Snap is available in the official [Snap store](https://snapcraft.io/store), as 
+[magma-access-gateway](https://snapcraft.io/magma-access-gateway). <br>
 This Snap is currently delivered as a [classic](https://snapcraft.io/docs/snap-confinement) Snap.
 
-**Required software:**
+**Installing Magma AGW snap:**
 
-- [Snapcraft](https://snapcraft.io/docs/snapcraft-overview)
-- [Multipass](https://multipass.run/)
-- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+Installation progress can be monitored at all times using `journalctl`.
 
-**Building Magma AGW snap:**
+To install [magma-access-gateway](https://snapcraft.io/magma-access-gateway) execute:
 
-1. Install required software listed above
-2. Checkout [magma-access-gateway-snap](https://github.com/canonical/magma-access-gateway-snap)
-3. Inside checked out repository's main directory execute:<br>
-   `snapcraft`<br>
-   To see what's happening during Snap's building process, `-d` can be used along with above 
-   command.
+```bash
+snap install magma-access-gateway --classic
+```
 
-**Installing Magma AGW snap:**
+Once Snap is installed (it will be indicated by the `magma-access-gateway 1.6.0 installed`
+message printed to the console), start AGW deployment by executing:
+
+```bash
+magma-access-gateway.install
+```
 
-1. Copy snap to AGW host machine.
-2. On AGW host machine, as `root` execute<br>
-   `snap install ${PATH_TO_THE_SNAP_FILE} --dangerous --classic`<br>
-   Since Snap has been built locally, `--dangerous` flag is required to deploy it.
-3. Once Snap is installed (it will be indicated by the `magma-access-gateway 1.6.0 installed`
-   message printed to the console), start AGW deployment by executing:<br>
-   - `magma-access-gateway` - to deploy Magma AGW using DHCP configured SGi interface
-   - `magma-access-gateway --ip-address 1.1.1.1/24 --gw-ip-address 1.1.1.1` - to deploy Magma AGW
-     using statically configured SGi interface
+[magma-access-gateway](https://snapcraft.io/magma-access-gateway) installer exposes various
+deployment configuration parameters. To see the list of currently supported parameters along with
+their descriptions, execute:
 
-   Magma AGW uses **Google (8.8.8.8)** and **OpenDNS (208.67.222.222)** DNS servers by default. 
-   DNS servers can be configured by using `--dns` flag along with `magma-access-gateway` 
-   command. Custom DNS servers should be specified as space separated list of IP addresses.
+```bash
+magma-access-gateway.install --help
+```
```

Comment:
Shouldn't we move this section below "**Configuring Magma Access Gateway:**"?

Reply:
No, DNS is configured upon installation. Configuration section relates to `magma-access-gateway.configure` command.